index.js

```diff
@@ -16,15 +16,17 @@ SparqlHttp.prototype.getQuery = function (query, options) {
   var url = null
 
   if (!options.update) {
-    url = (options.endpointUrl || this.endpointUrl) + '?query=' + encodeURIComponent(query)
+    url = new URL(options.endpointUrl || this.endpointUrl)
+    url.searchParams.append('query', query)
   } else {
-    url = (options.updateUrl || this.updateUrl) + '?update=' + encodeURIComponent(query)
+    url = new URL(options.updateUrl || this.updateUrl)
+    url.searchParams.append('update', query)
   }
 
   options.method = 'get'
   options.headers['Accept'] = options.headers['Accept'] || options.accept
 
-  return this.fetch(url, options)
+  return this.fetch(url.toString().replace(/\+/g, '%20'), options)
```

Comment:
This one is rather weird. The `URLSearchParams#append` method like to use the plus symbol instead of escaping spaces as `%20`. Both are equal and I'd rather get rid of the replace and change the tests but this was less impactful in terms of a single PR

Reply:
I don't know of any store which doesn't support `+`.

@ktk any idea if triplestores care about `%20` vs. `+` escaping for spaces?

Reply:
@bergos not that I know of but I did not really test that so far